test-case/check-suspend-resume-with-audio.sh

```diff
@@ -98,13 +98,27 @@ do
     sleep 1
     # check process status is correct
     sof-process-state.sh $process_id
-    [[ $? -ne 0 ]] && dloge "$cmd error without process" && exit 1
+    if [ $? -ne 0 ]; then
+        dloge "$cmd error without process"
```

Comment:
`$cmd error without process`, this info is misleading, do you mean : we run $cmd, but there is no such process named $cmd? if so, you can change this info to :: `error: no such process named $cmd` or `error: no such process for $cmd` or something like this.